sdk/cosmos/azure-cosmos/docs/Metrics.md

```diff
@@ -0,0 +1,174 @@
+        .build();
+
+    consoleReporter.start(1, TimeUnit.SECONDS);
+
+    DropwizardConfig dropwizardConfig = new DropwizardConfig() {
+
+        @Override
+        public String get(@Nullable String key) {
+            return null;
+        }
+
+        @Override
+        public String prefix() {
+            return "console";
+        }
+
+    };
+
+    final DropwizardMeterRegistry consoleLoggingRegistry = new DropwizardMeterRegistry(
+        dropwizardConfig, dropwizardRegistry, HierarchicalNameMapper.DEFAULT, Clock.SYSTEM) {
+        @Override
+        protected Double nullGaugeValue() {
+            return Double.NaN;
+        }
+
+        @Override
+        public void close() {
+            super.close();
+            consoleReporter.stop();
+            consoleReporter.close();
+        }
+    };
+
+    consoleLoggingRegistry.config().namingConvention(NamingConvention.dot);
+    return consoleLoggingRegistry;
+}
+```
+
+The micrometer.io documentation has a list with samples on how to create a `MeterRegistry` for several telemetry systems: `https://micrometer.io/docs`. You can also find an implementation via an Azure Monitor `MeterRegistry` here: [Spark sample with Azure Monitor `MeterRegistry`](https://aka.ms/azure-cosmos-spark_CosmosMetricsApplicationInsightsPlugin)
+
+
+
+## Which tags/dimensions are associated with metrics?
+
+| Tag/dimension name                   | Sample value                                                 | Description                                                  | Scope                       |
+| ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | --------------------------- |
+| `Container`                          | `accountName / databaseName / containerName`                 | The fully qualified container name                           | operations + requests       |
+| `Operation`                          | `Document / ReadFeed` or `Document / queryItems / SomeLogicalQueryIdentifier` | The operation type and for queries with optional logical operation identifier as suffix | operations + requests       |
+| `OperationStatusCode`                | `200` or `429` etc.                                          | The status code of the operation reported to the app/svc (could indicate sucess `200` even after hitting errors and retyring successfully) | operations  + requests      |
+| `ClientCorrelationId`                | `MyClientUsingAADAuth`                                       | An identifier of the Cosmos client instance - can be specified via the `CosmosClientBuilder. clientTelemetryConfig(). clientCorrelationId(String)` method or gets auto-generated | operations + requests       |
+| `ConsistencyLevel`                   | `Eventual`, `BoundedStaleness`, `Strong` or `Session`        | The consistency level used for the operation                 | operations + requests       |
```

Comment:
missed consistencyPrefix

Reply:
Hmm - yeah I left it out intentionally because it is effectively same as eventual (so kind of useless)- but you are right it is possible to show up.

Reply:
Fixed